### PR TITLE
Move Amount to ProductMeasurement

### PR DESCRIPTION
**Pull request by skearnes**
Closes #516 

Once the proto changes are approved, I'll update the tests.

### CODE REVIEW

File: ord_schema/proto/reaction.proto [hunk 1]
```diff
@@ -911,6 +909,8 @@ message ProductMeasurement {
     COUNTS = 7;
     // Intensity, e.g., of a TLC spot, ELSD, or GC FID.
     INTENSITY = 8;
+    // Quantitative amount of this product, usually mass.
```
Comment (connorcoley) [inline ord_schema/proto/reaction.proto:912]: "Quantitative amount of this product, **as defined in the 'amount' field**, usually as a mass."?

**Review by connorcoley — COMMENTED** [review]
Proto changes seem good to me.

File: ord_schema/proto/reaction.proto (hunk 1 above)
Comment (skearnes) [reply ord_schema/proto/reaction.proto]: Done.

**Review by connorcoley — APPROVED** [review]

**Comment by skearnes** [comment]
@michaelmaser any comments?

**Review by michaelmaser — APPROVED** [review]
LGTM!

**Pull request merged**